.github/workflows/deploy-docs-site.yml

```diff
@@ -83,3 +83,31 @@ jobs:
             {\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"_merged by_: *${{ github.actor }}*\"}},
             {\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"<https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}|View Action Run>\"}}
             ]}]}
+
+
+  set-instatus-incident-on-failure:
+    name: Create Instatus Incident on Failure
+    runs-on: ubuntu-latest
+    needs:
+      - build-and-deploy-docs
+    if: ${{ failure() && github.ref == 'refs/heads/master' }}
+    steps:
+      - name: Call Instatus Webhook
+        uses: joelwmale/webhook-action@master
+        with:
+            url: ${{ secrets.INSTATUS_CONNECTOR_CI_WEBHOOK_URL }}
+            body: '{ "trigger": "down" }'
```

Comment:
That's a rather reasonable payload :D

Reply:
Is instastatus smart enough to know which named service the incident belongs to based on the action name or something like that?  Or, since we are using the same INSTATUS_CONNECTOR_CI_WEBHOOK_URL, are these incidents all in the same service?

Reply:
> Could not find documentation for passing any type of arguments to a instatus template. So we cannot specify what failed in the incident

Maybe... multiple URLS?

Reply:
Unfortunately small payload means much less context.

So the webhook url (and as a result the instatus template) is the area that "knows" what service this relates too.

This means that we have two instatus services
* Platform Critical Github Actions
* Connector Critical Github Actions

which each have a unique template

and each of those templates have a unique webhook url

One of them is defined in our secrets as `INSTATUS_CONNECTOR_CI_WEBHOOK_URL` and the other `INSTATUS_PLATFORM_CI_WEBHOOK_URL`